__tests__/user/user.query.test.js

```diff
@@ -32,61 +71,19 @@ describe('queries', () => {
                         email
                         firstName
                         lastName
-                        phoneNumber
```

Comment:
Revert this delete, you should get all possible fields and check all of them

Reply:
fixed